store/map/index.js

```diff
@@ -193,6 +212,9 @@ export const getters = {
   getCollapsedDatasets (state) {
     return state.collapsedDatasets
   },
+  getLoadingState (state) {
+    return state.loadingRasterLayers
```

Comment:
Why do we need these getters? They don't do anything....

Reply:
consistency with previous work? You suggest just using $store.state.loadingRasterLayers for example.

Need to discuss these choices for next sprint maybe

Reply:
I thought this was in a component. In the store it's the  normal way to do it.